Userland/Services/WindowServer/Menu.cpp

```diff
@@ -664,6 +669,7 @@ void Menu::do_popup(Gfx::IntPoint position, bool make_input, bool as_submenu)
 
     constexpr auto margin = 10;
     Gfx::IntPoint adjusted_pos = m_unadjusted_position = position;
+    m_flipped_at_vertical_axis = false;
 
     if (adjusted_pos.x() + window.width() >= screen.rect().right() - margin) {
         // Vertically translate the window by its full width, i.e. flip it at its vertical axis.
```

Comment:
The comment says `// Vertically translate` and the next line translates it horizontally afaict.

I find the `m_flipped_at_vertical_axis` name confusing. It comes from this comment, which I also find confusing. It's not flipped, it's shifted to the left. Maybe this could be called `m_opens_to_the_left`?